WWDC/SessionsTableViewController.swift

```diff
@@ -205,15 +203,21 @@ class SessionsTableViewController: NSViewController {
         searchController.searchField.isEnabled = true
     }
 
+    /// Variable used to track whether the initial display of rows has occurred
+    /// - warning: Not meant to be used outside of `performFirstUpdateIfNeeded`
+    private var setupDone = false
+
+    /// This function is meant to ensure the table view gets populated
+    /// even if its data model gets added while it is offscreen. Specifically,
+    /// when this table view is not the initial active tab.
     private func performFirstUpdateIfNeeded() {
+        guard !setupDone else { return }
+        setupDone = true
+
         switch style {
         case .schedule:
             updateScheduleList()
```

Comment:
Notice the missing guard here. This bug was being hidden by the appearance method bug because generally outside of WWDC week, the schedule tab is not the saved tab. But if the schedule tab is the saved tab, this bug presented itself by reloading the table every time you would visit the schedule tab.